libs/model/src/main/java/de/cyface/model/TrackBucket.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2021-2022 Cyface GmbH
+ *
+ * This file is part of the Serialization.
+ *
+ * The Serialization is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Serialization is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the Serialization. If not, see <http://www.gnu.org/licenses/>.
+ */
+package de.cyface.model;
+
+import java.util.Date;
+
+/**
+ * The mongo database "bucket" which contains a slice of a track.
+ *
+ * @author Armin Schnabel
+ * @since 1.5.0
+ * @version 1.0.0
+ */
+@SuppressWarnings("unused") // Part of the API
+public class TrackBucket {
```

Comment:
moved to libs/model from collector/libs/api as this is now also required by backend/libs/mongo-reader (to read deserialized data, TrackBuckets, in the surface pipelines)